tests/integration/components/export-tool-test.js

```diff
@@ -0,0 +1,215 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+import ispSurvey from '../../fixtures/isp-survey';
+
+moduleForComponent('export-tool', 'Integration | Component | export tool', {
+    integration: true
+});
+
+/**
+ * Adds double quotes around a string
+ *
+ * @param value {*} - The value to quote
+ * @returns {string}
+ */
+function quote(value) {
+    if (typeof value !== 'string') {
+        return value;
+    }
+
+    return `"${value}"`;
+}
+
+/**
+ * A legible way to define a *SV file in js
+ *
+ * @param {Array<(Array<String>|String)>} arr An array of rows
+ * @param {String} columnSeparator The column separator (defaults to ',')
+ * @returns string
+ */
+function svJoin(arr, columnSeparator) {
+    return arr
+        .map((item, i) =>
+            Array.isArray(item) ?
+                (i > 0 ? item.map(quote) : item).join(columnSeparator) :
+                item
+        )
+        .join('\n');
```

Comment:
I noticed that also and meant to bring it up. I haven't found any documentation of this, but it may just be an OS thing--really don't know why it works this way. Maybe we can mitigate this by testing the blob in the download.